gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -271,7 +271,7 @@ def gard_mondo_mapping_status(
             # 'object_label': '',
         }]
         robot_df = pd.concat([pd.DataFrame(robot_subheader), robot_df])[list(robot_subheader[0].keys())]
-        robot_df.to_csv(RELEASE_DIR / f'mondo-gard{file_suffix}.robot.template.tsv', index=False)
+        robot_df.to_csv(RELEASE_DIR / f'mondo-gard{file_suffix}.robot.template.tsv', index=False, sep='\t')
```

Comment:
the bugfix

Reply:
Added to new release: https://github.com/monarch-initiative/gard/releases/tag/v2023-08-14